link/src/main/java/com/stripe/android/link/ui/signup/SignUpScreen.kt

```diff
@@ -195,11 +199,14 @@ private fun PhoneCollectionSection() {
             style = MaterialTheme.typography.caption
         )
         Button(
```

Comment:
It feels a little weird that the button is inside of the PhoneCollectionSection.   Can we move it out to the SignUpScreen?  

The design paradigms of the email handling and phone number handling feel quite different.  I wonder if converting the PhoneCollection into something more similar to email will help keep them more consistent.  Happy to talk this through in a phone call too.